src/controllers/productController.ts

```diff
@@ -107,7 +109,19 @@ const createProduct: RequestHandler = async (req, res, next) => {
 const patchProduct: RequestHandler = async (req, res, next) => {
   const { id } = req.params;
   try {
+    const oldProduct = await ProductService.getProduct(id);
     const product = await ProductService.patchProduct(id, req.body);
+    if (product.price !== oldProduct.price) {
+      const userList = await ProductService.getProductLikeUserList(id);
+      userList.forEach(async user => {
+        const notification = await NotificationService.createNotification({
+          userId: user.userId,
+          type: 'LIKE',
+          message: `상품 ${oldProduct.name}의 가격이 ${oldProduct.price}에서 ${product.price}로 변경되었습니다.`,
+        });
+        eventBus.emit('newNotification', notification);
```

Comment:
emit 할 때 사용되는 이벤트 호출명을 constants 로 빼서 사용하면 호출 되는 부분을 쉽게 찾을 수 있어, 코드 유지보수성이 올라갑니다!
```suggestion
        eventBus.emit('newNotification', notification);
```